<!doctype html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>What Uniparental Disomy Tells Us About Improper Imprinting in Humans - Tassilo Neubauer</title>

    <!-- Open Graph Meta Tags -->
    <meta property="og:title" content="What Uniparental Disomy Tells Us About Improper Imprinting in Humans" />
    <meta property="og:type" content="article" />
    <meta property="og:url" content="https://www.tassiloneubauer.com/uniparental-disomy.html" />
    <meta property="og:description" content=""An exploration of genetic imprinting disorders and their implications for germline engineering"" />
    <meta property="og:site_name" content="Tassilo Neubauer" />
    
    <meta property="og:image" content="https://www.tassiloneubauer.com/images/diagram_upd.jpg" />
    

    <link rel="stylesheet" href="/styles.css" />
    <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.24.1/prism.min.js"></script>
    <link
      rel="stylesheet"
      href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.24.1/themes/prism.min.css"
    />
    <link
      rel="alternate"
      type="application/rss+xml"
      title=""
      href="/feed.xml"
    />
  </head>
  <body>
    <header>
  <nav class="container">
    <h1><a href="/">Tassilo Neubauer</a></h1>
    <!-- <ul> -->
    <!--     <li><a href="/">Home</a></li> -->
    <!--     <li><a href="/about.html">About</a></li> -->
    <!--     <li><a href="/archive.html">Archive</a></li> -->
    <!-- </ul> -->
  </nav>
</header>

    <main class="container">
      <article
        class="blog-post"
      >
        <h2>What Uniparental Disomy Tells Us About Improper Imprinting in Humans</h2>
        <time datetime="2025-03-28T00:00:00"
          >Published on: March 28, 2025</time
        >
        <div class="post-content"><h1>TLDR</h1>
<p><strong>We do not yet understand all the genetic imprints that play an important role in early embryonic development. Known imprinting disorders (</strong><a href="https://en.wikipedia.org/wiki/Prader%E2%80%93Willi_syndrome"><strong>Prader-Willi</strong></a><strong>,</strong> <a href="https://en.wikipedia.org/wiki/Angelman_syndrome"><strong>Angelman</strong></a><strong>, etc.) however, do seem to cover all the most detrimental imprints, since for all cases of uniparental disomy (having 1 pair of chromosomes from only one parent), we know of either the imprinting disorder or of a phenotypically normal case. One exception is maternal UPD 19, which might be rare</strong> <strong><em>because</em></strong> <strong>proper imprinting on chromosome 19 is crucial for early development in humans.</strong>
Imprinting and germline engineering
===================================
Most strong methods of germline engineering <a href="https://www.lesswrong.com/posts/2w6hjptanQ3cDyDw7/methods-for-strong-human-germline-engineering">require us to know what the proper epigenomics of an embryo look like</a>. Measuring the epigenome is quite tricky. Maintaining specifically imprinting is even more tricky. We currently have no method of culturing naive embryonic stem cells that does not lead to loss of imprinting.<sup id="fnref1"><a href="#fn1">1</a></sup> While there is a lot of literature on imprinting and experiments in mice to determine the mechanisms behind known imprints, research in this area does not happen with the explicit goal of germline engineering. There is no comprehensive list of all human imprints.<sup id="fnref2"><a href="#fn2">2</a></sup> So there might be regions whose importance for germline engineering in humans might be overlooked if there is no known associated disorder in humans. Hence, this post. 
This post assumes basic familiarity with epigenetics. The next paragraph is a 1-paragraph summary, but I recommend reading <a href="https://denovo.substack.com/p/what-is-epigenetics">these</a> <a href="https://denovo.substack.com/p/epigenetics-of-the-mammalian-germline">two</a> excellent posts on epigenetics. If you know what DNA methylation and imprinting are, you can skip the posts.
The short explanation is that DNA methylations are small chemical marks on the DNA that change which proteins bind well to the DNA. Together with other chemical marks on <a href="https://en.wikipedia.org/wiki/Histone">histones</a>, which are spindles that DNA is rolled around, these marks are how a cell knows that it is a skin cell, rather than a neuron. Imprinting is a phenomenon where the two sets of chromosomes in early mammal embryos behave differently depending on if they were inherited from the father or the mother side. Imprinting appears to be mostly due to differences in DNA methylation and histone marks.
Imprinting disorders and uniparental disomy
===========================================
One of the earliest ways we discovered imprinting is through disorders like <a href="https://en.wikipedia.org/wiki/Prader%E2%80%93Willi_syndrome">Prader-Willi syndrome</a>, which had already been linked to deletions of some genes on Chromosome 15. Some patients though did not have any such deletions. Instead, they had inherited both of their homologous chromosomes 15 from their mother. This is usually called maternal uniparental disomy 15 (UPD). Uniparental disomies can develop through a few different mechanisms that I'll explain below.
What is uniparental disomy, and how does it develop?</p>
<hr />
<p>Uniparental disomy means someone has a chromosome twice from the same parent, rather than 1 copy from each parent. In principle, this is an excellent way to identify problems that could be caused by imprinting. Unfortunately, of course, there are additional confounders other than imprinting on the chromosome in UPD. In most cases of UPD, all or parts of the two chromosomes are homozygous. They are respectively called isoUPD and partial isoUPD. This often leads to deleterious recessive genes surfacing, just like with incest. 
20% of the time<sup id="fnref3"><a href="#fn3">3</a></sup> however, UPD comes with heterozygous chromosomes, so the affected chromosomes are just the same as those of one of the parents (or the same chromosomes, but crossed over). Even in those cases, UPD is still not a perfect "treatment group" for the effects of imprinting, since UPDs are most often the result of "rescued" trisomies: trisomies that get accidentally corrected by an error during chromosome segregation, turning one cell disomic and the other one tetrasomic for that specific chromosome. The more healthy disomic cells then outcompete the aneuploid cell, such that the embryo is completely or mostly made up of euploid cells (aneuploid cells tend to form the placenta instead). Analogously, UPD can be the product of a rescued monosomy.<sup id="fnref4"><a href="#fn4">4</a></sup> In practice, this means, that when looking at a cohort of UPD patients, any symptoms they have in common could also be due to trisomy or monosomy in the placenta or even in some tissues of the patient.<sup id="fnref5"><a href="#fn5">5</a></sup>
<img alt="https://www.tassiloneubauer.com/images/diagram_upd.jpg" src="https://www.tassiloneubauer.com/images/diagram_upd.jpg" />
UPD resulting from trisomie illustrated. The image is an edited version of Figure 1 in  <a href="https://linkinghub.elsevier.com/retrieve/pii/S0002-9297(19)30356-8">(Nakka et al., 2019)</a>
More on known imprinting disorders</p>
<hr />
<p>Most imprints seem to have evolved due to parental conflict about how much effort to invest in particular offspring in non-monogamous species.<sup id="fnref6"><a href="#fn6">6</a></sup><sup id="fnref7"><a href="#fn7">7</a></sup> If the father and mother favour different levels for the expression of a gene and the respective benefits outweigh the benefits of expressing the gene from both chromosomes, then the gene evolves to become imprinted and is only expressed from the side that favours its expression.<sup id="fnref7-2"><a href="#fn7">7</a></sup>
This also means every imprinting disorder like Prader-Willi syndrome (caused by maternal UPD) has a sibling syndrome that is caused by inheriting both chromosomes 15 from one's father (paternal UPD): <a href="https://en.wikipedia.org/wiki/Angelman_syndrome">Angelmann syndrome</a>.
Through UPD, we later discovered imprinting for a few other chromosomes. Most notably, <a href="https://en.wikipedia.org/wiki/Silver%E2%80%93Russell_syndrome">Silver-Russel syndrome</a> can be caused by maternal UPD on chromosomes 7, 11 or 20. Paternal UPD11 leads to <a href="https://en.wikipedia.org/wiki/Beckwith%E2%80%93Wiedemann_syndrome">Beckwith-Wiedemann syndrome</a>. Maternal UPD14 leads to <a href="https://en.wikipedia.org/wiki/Temple_syndrome">Temple syndrome</a>, while Paternal UPD14 leads to <a href="https://en.wikipedia.org/wiki/Kagami-Ogata_Syndrome">Kagami-Ogata syndrome</a>. Paternal UPD 6 leads to <a href="https://en.wikipedia.org/wiki/Transient_neonatal_diabetes">transient neonatal diabetes mellitus</a>. Paternal UPD20 leads to <a href="https://en.wikipedia.org/wiki/Pseudohypoparathyroidism">Pseudohypoparathyroidism type Ib</a>. 
In line with the parental conflict theory of imprinting, when I google the symptoms and first-line treatments for these disorders, maternal UPD is associated with hypoglycemia and patients tend to benefit from growth hormone treatments. Meanwhile, paternal UPD is associated with hyperglycemia and lines of treatment seem more varied. Loss of imprinting is also common in some kinds of cancers,<sup id="fnref8"><a href="#fn8">8</a></sup> so the corresponding types of UPD probably make it more likely to get cancer.
All of these disorders can also be caused by genetic deletions, which allowed us to identify where the imprints for these disorders are located. Except for maternal UPD7, where so far a specific gene or region hasn't been identified.<sup id="fnref9"><a href="#fn9">9</a></sup> UPD16 sometimes leads to Silver-Russel symptoms, but it is unclear if this is just due to mosaic trisomy 16 or other confounders.<sup id="fnref9-2"><a href="#fn9">9</a></sup>
It turns out, that after these more common cases, uniparental disomy cases are a lot less often reported.
<img alt="published UPD cases" src="https://www.tassiloneubauer.com/images/published_upd_cases.png" />
Published UPD cases by parental origin and chromosome. Figure S6 from <a href="https://linkinghub.elsevier.com/retrieve/pii/S0002-9297(19)30356-8">(Nakka et al., 2019)</a>
What is up with the underreported UPD cases?</p>
<hr />
<p>One question was still unclear to me after looking up all the known imprinting disorders: To what extent do we even know all the "lethal" or highly detrimental imprinting regions? By highly detrimental, I mean something that would not go undetected even if the disease is rare. For example, it could be that one of the rare uniparental disomy cases is so deadly already in early pregnancy that we never observed any cases. We would still have caught any such imprinting regions if they also exist in mice, but for any important imprints missing in mice or if they happen to be less of a problem in mice, UPD would be one of the very few ways we could observe them.
It turns out it is <em>probably</em> the latter (imprinting is more benign for the other cases of UPD, although maternal UPD19 might be an exception). There are two main sources of evidence making me believe this.
The first one is this website maintained by Thomas Liehr who fairly comprehensively gathered together most (all?) <a href="https://cs-tl.de/DB/CA/UPD/0-Start.html">existing case studies of UPD</a>. I manually went through all the database entries and for all chromosomes except chromosome 5, I was able to find full UPD cases that were "phenotypically normal" for either paternal or maternal UPD. From looking over <a href="https://cs-tl.de/DB/CA/UPD/5-UPDm.html">the cases</a> <a href="https://cs-tl.de/DB/CA/UPD/5-UPDp.html">of</a> <a href="https://cs-tl.de/DB/CA/UPD/5-UPDa.html">chromosome 5</a> my impression from the variedness of symptoms was that this was more likely due to recessive genetic disorders and mosaic trisomy 5 rather than due to imprinting.
There are some chromosomes, like Chromosomes 19, where we do have <a href="https://www.sciencedirect.com/science/article/pii/S1028455921000231?via%3Dihub">1 case of UPD19</a> where the child didn't have apparent problems. We do have evidence that there are a few imprints on chromosome 19 (specifically an imprinting cluster in 19q13.4–19q13.43), whose homologs were first discovered through UPD studies in mice.<sup id="fnref10"><a href="#fn10">10</a></sup> So there are very likely effects from incorrect imprinting on chromosome 19, though they might be small. Also, while for some chromosomes we have phenotypically normal data for adults (among others <a href="https://pubmed.ncbi.nlm.nih.gov/9758608/">chr1,</a> chr21, chr22), for others we only have this reported for newborns. 
The second line of evidence for less known uniparental disomy cases being benign, comes from this large data analysis of 23andMe's and UK Biobanks data, showing that a lot of these lesser-known imprinting disorders aren't rarer than the commonly reported ones:
<img alt="plot " src="https://www.tassiloneubauer.com/images/UPD_cases_23andMe_UK_Biobank.png" />
Cases of UPD found in 23andMe's and UK Biobank through finding large homozygous regions (runs of homozygosity or ROH). Screenshot from Figure 3 from <a href="https://linkinghub.elsevier.com/retrieve/pii/S0002-9297(19)30356-8">(Nakka et al., 2019)</a>
23andMe's dataset has enough parent-child pairs, that we can even get a breakdown by paternal or maternal origin:
<img alt="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/de65/6848996/523b8ccbbf51/gr2.jpg" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/de65/6848996/523b8ccbbf51/gr2.jpg" />
UPD cases in ~1 million parent child pairs from <a href="https://linkinghub.elsevier.com/retrieve/pii/S0002-9297(19)30356-8">(Nakka et al., 2019)</a>
If they are this common, we would know the phenotype of UPD5 by now, just as we know it for UPD7, if there was a clear phenotype. 
Let's look at the cases, where we don't have maternal and paternal cases in the figure above. In principle, all the above data still makes it possible that maternal UPD19 is quite bad for embryos if all the UPD19 cases in <a href="https://linkinghub.elsevier.com/retrieve/pii/S0002-9297(19)30356-8">(Nakka et al., 2019)</a> turn out to be paternal UPD19. To my knowledge, there is no reported case of maternal UPD19. So perhaps that imprinting region on chromosome 19 is quite important. Or some other region makes this rare ¯\<em>(ツ)</em>/¯ (though the absence of any case studies is not necessarily that much evidence that maternal UPD19 is deadly, since trisomy 19 is known to be deadly early on for embryos). UPD18 is not a problem here, since there are documented phenotypically normal cases for both maternal and paternal UPD18.<sup id="fnref11"><a href="#fn11">11</a></sup> Both maternal and paternal UPD11 and UPD20 are known to be imprinted, as mentioned earlier.
Conclusion
==========
So overall, while we do not have a complete understanding of all imprints, we have <em>probably</em> identified the most detrimental ones. Exceptions are UPD7, where we do not know which gene(s) causes Silver-Russel syndrome. We are also still in the dark about maternal UPD19, which might be deadly to embryos.
<em>Thanks to Tsvi Benson-Tilsen for discussion and</em> <a href="https://www.lesswrong.com/users/genesmith?from=search_autocomplete"><em>Gene Smith</em></a>  <em>for employing me to do this research.</em></p>
<div class="footnote">
<hr />
<ol>
<li id="fn:bzszbf4ybw">
<p>Bar, S., Schachter, M., Eldar-Geva, T., &amp; Benvenisty, N. (2017). Large-Scale Analysis of Loss of Imprinting in Human Pluripotent Stem Cells. <em>Cell Reports, 19</em>(5), 957–968. https://doi.org/10.1016/j.celrep.2017.04.020&#160;<a class="footnote-backref" href="#fnref:bzszbf4ybw" title="Jump back to footnote 1 in the text">&#8617;</a></p>
</li>
<li id="fn:s1t55u1o35c">
<p>The following list is often cited, but a lot of imprints in it are only "predicted": Geneimprint: Genes. (n.d.). Retrieved March 28, 2025, from https://www.geneimprint.com/site/genes-by-species&#160;<a class="footnote-backref" href="#fnref:s1t55u1o35c" title="Jump back to footnote 2 in the text">&#8617;</a></p>
</li>
<li id="fn:srvplo600zm">
<p>See table 1 in <sup id="fnref:bhs0c4rfhbc"><a class="footnote-ref" href="#fn:bhs0c4rfhbc">12</a></sup>.&#160;<a class="footnote-backref" href="#fnref:srvplo600zm" title="Jump back to footnote 3 in the text">&#8617;</a></p>
</li>
<li id="fn:95m152lpgf5">
<p>Rudd, M. K., Schleede, J. B., Williams, S. R., Lee, K., Laffin, J., Pasion, R., &amp; Papenhausen, P. R. (2018). Monosomy X rescue explains discordant NIPT results and leads to uniparental isodisomy. <em>Prenatal Diagnosis, 38</em>(12), 920–923. https://doi.org/10.1002/pd.5349&#160;<a class="footnote-backref" href="#fnref:95m152lpgf5" title="Jump back to footnote 4 in the text">&#8617;</a></p>
</li>
<li id="fn:kvttz6w51ip">
<p>Another way UPD can develop is if sperm and eggs are accidentally disomic and nullisomic in the same chromosome, cancelling out each other's mistakes (see <a href="https://doi.org/10.1002/(SICI)1521-1878(200005)22:5&lt;452::AID-BIES7&gt;3.0.CO;2-K">Robinson, 2000</a>). If we get such an accidental UPD case that also happens to be a heterodisomy (presumably 50% of these cases are heterodisomy?) that would be quite close to isolating the effects just due to imprinting and not due to any mosaic trisomy in either the embryo or the placenta. Unfortunately this is not going to happen often. UPD at conception is rare for most chromosomes except for those that frequently mis-segregate, like the X-chromosome (which is aneuploid in ~0.5% of healthy sperm; <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1914964/">Spriggs et al., 1996</a>). I couldn't find really good numbers on oocytes, but they tend to be aneuploid more frequently — somewhere between 3–10× more often than sperm.&#160;<a class="footnote-backref" href="#fnref:kvttz6w51ip" title="Jump back to footnote 5 in the text">&#8617;</a></p>
</li>
<li id="fn:2cys0cpjj4s">
<p>There are a lot of other theories how imprinting evolved, but the consensus seems correct on this one. Read this one for more: Spencer, H. G., &amp; Clark, A. G. (2014). Non-conflict theories for the evolution of genomic imprinting. <em>Heredity, 113</em>(2), 112–118. https://doi.org/10.1038/hdy.2013.129&#160;<a class="footnote-backref" href="#fnref:2cys0cpjj4s" title="Jump back to footnote 6 in the text">&#8617;</a></p>
</li>
<li id="fn:kpm0f9n1u8c">
<p>Moore, T., &amp; Haig, D. (1991). Genomic imprinting in mammalian development: A parental tug-of-war. <em>Trends in Genetics, 7</em>(2), 45–49. https://doi.org/10.1016/0168-9525(91)90230-N&#160;<a class="footnote-backref" href="#fnref:kpm0f9n1u8c" title="Jump back to footnote 7 in the text">&#8617;</a></p>
</li>
<li id="fn:0v1tdhdbiytm">
<p>Jelinic, P., &amp; Shaw, P. (2007). Loss of imprinting and cancer. <em>The Journal of Pathology, 211</em>(3), 261–268. https://doi.org/10.1002/path.2116&#160;<a class="footnote-backref" href="#fnref:0v1tdhdbiytm" title="Jump back to footnote 8 in the text">&#8617;</a></p>
</li>
<li id="fn:64stc14kae">
<p>Kurup, U., Lim, D. B. N., Palau, H., Maharaj, A. V., Ishida, M., Davies, J. H., &amp; Storr, H. L. (2024). Approach to the Patient With Suspected Silver-Russell Syndrome. <em>The Journal of Clinical Endocrinology and Metabolism, 109</em>(10), e1889–e1901. https://doi.org/10.1210/clinem/dgae423&#160;<a class="footnote-backref" href="#fnref:64stc14kae" title="Jump back to footnote 9 in the text">&#8617;</a></p>
</li>
<li id="fn:xm7gkvpix6">
<p>Access via Internet Archive: Beechey CV, Cattanach BM, Blake A, Peters J (2008), MRC Mammalian Genetics Unit, Harwell, Oxfordshire. World Wide Web Site – Mouse Imprinting Data and References – http://www.har.mrc.ac.uk/research/genomic_imprinting&#160;<a class="footnote-backref" href="#fnref:xm7gkvpix6" title="Jump back to footnote 10 in the text">&#8617;</a></p>
</li>
<li id="fn:61o5wzr6edv">
<p>See https://cs-tl.de/DB/CA/UPD/18-UPDp.html and https://cs-tl.de/DB/CA/UPD/18-UPDm.html&#160;<a class="footnote-backref" href="#fnref:61o5wzr6edv" title="Jump back to footnote 11 in the text">&#8617;</a></p>
</li>
<li id="fn:bhs0c4rfhbc">
<p>Nakka, P., Smith, S. P., O’Donnell-Luria, A. H., McManus, K. F., Agee, M., Auton, A., Bell, R. K., Bryc, K., Elson, S. L., Fontanillas, P., Furlotte, N. A., Hicks, B., Hinds, D. A., Jewett, E. M., Jiang, Y., Lin, K.-H., McCreight, J. C., Huber, K. E., Kleinman, A., … Sathirapongsasuti, J. F. (2019). Characterization of Prevalence and Health Consequences of Uniparental Disomy in Four Million Individuals from the General Population. <em>The American Journal of Human Genetics, 105</em>(5), 921–932. https://doi.org/10.1016/j.ajhg.2019.09.016&#160;<a class="footnote-backref" href="#fnref:bhs0c4rfhbc" title="Jump back to footnote 12 in the text">&#8617;</a></p>
</li>
</ol>
</div></div>
      </article>
    </main>

    <footer>
    <div class="container">
        <p>© 2024 Tassilo Neubauer. Copy freely.</p>
    </div>
</footer>

    <script src="/script.js"></script>
  </body>
</html>